fastai/transforms.py

```diff
@@ -518,6 +538,30 @@ def __init__(self, n_holes, length, tfm_y=TfmType.NO):
     def do_transform(self, img, is_y):
         return cutout(img, self.n_holes, self.length)
 
+class GoogleNetResize(CoordTransform):
+    
+    def _init(self, params=None):
+        if params:
+            for k, v in params.items():
+                if k != 'self':
```

Comment:
Can you add a comment explaining what's happening here, and why?

Reply:
_init method was used to convert each element in the params dict to attribute of GoogleNetResize class without having to explicitly write for every argument. I have removed this as the parameters are now regular arguments.